src/components/RecipeSearch.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+import styled from 'styled-components';
+
+const StyledLink = styled(Link)`
+    float: right !important;
+    padding: 6px !important;
+    margin-top: 8px !important;
+    margin-right: 16px !important;
+    font-size: 17px !important;
+`;
+
+class RecipeSearch extends Component {
```

Comment:
- Why is this in `.js` and not in typescript? 
- avoid class components, you'll see that in our projects you'll be asked to always use functional components + hooks instead

Reply:
I missed that one when refactoring everything to TS on the last stage. Thanks, updated